pyccl/halos/massdef.py

```diff
@@ -78,33 +84,36 @@ class MassDef(CCLAutoreprObject):
         Delta (float): overdensity parameter. Pass 'vir' if using virial
             overdensity.
         rho_type (string): either 'critical' or 'matter'.
-        c_m_relation (function, optional): concentration-mass relation.
+        concentration (function, optional): concentration-mass relation.
             Provided as a `Concentration` object, or a string corresponding
             to one of the supported concentration-mass relations.
             If `None`, no c(M) relation will be attached to this mass
             definition (and hence one can't translate into other definitions).
     """
     __repr_attrs__ = ("name",)
+    __getattr__ = deprecate_attr(pairs=[('c_m_relation', 'concentration')]
+                                 )(super.__getattribute__)
 
-    def __init__(self, Delta, rho_type, c_m_relation=None):
+    @warn_api
```

Comment:
shouldn't you be declaring the old name here?

Reply:
good catch, fixed!